ui/webui/resources/wasm/opaque_ke/index.js

```diff
@@ -0,0 +1,7 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { Login, Registration } from 'gen/brave/ui/webui/resources/wasm/opaque_ke/bundler/pkg'
+export { Login, Registration }
```

Comment:
nit: could these two lines be combined? i.e. `export { Login, Registration } from 'gen/brave/ui/webui/resources/wasm/opaque_ke/bundler/pkg'`

Reply:
Yup!